### ISSUE TITLE
Notice: Locking `master` branch

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/datawizard#81]
@easystats/maintainers Please take note that starting today midnight (CET), I will be locking the `master` branch, as noted here:
https://github.com/easystats/easystats/issues/177 

I will also add the following branch protection rule for the `master` branch:

![image](https://user-images.githubusercontent.com/11330453/156742695-b0ca9927-8488-40e1-bd75-46a93cf8fbc5.png)

That is, you can merge into `master` only after all checks pass and the PR has been reviewed and approved by at least one other maintainer.

Of course, this is necessary only if you are adding new features or making significant changes, but not for minor things like spell check, styling, etc., although it'd be appreciated if you do make PRs even in that context.

As organization members, you will have the privileges to push directly to `master` even if branches are protected, but I'd prefer if you don't do that.

**Comment by IndrajeetPatil** [comment]
Needless to say, the same rule applies to me 🙃

**Comment by DominiqueMakowski** [comment]
we have tried by the past of having a more programmer-like workflow with a dev branch that gets merged into master, but it never really worked out... I'm worried we don't have the discipline for that 🥲

**Comment by DominiqueMakowski** [comment]
Ah you mean only for datawizard for now, I thought directly for all the packages ^^

**Comment by strengejacke** [comment]
Even me started to use PRs more often in the past ;-)

**Comment by strengejacke** [comment]
I hope you didn't plan this because of https://github.com/easystats/datawizard/commits/master ?

**Comment by strengejacke** [comment]
And Indra like:

https://user-images.githubusercontent.com/26301769/156748881-d4f5c995-f85b-4a95-b45b-df5f64077c0a.mp4
